.github/workflows/trivy.yml

```diff
@@ -0,0 +1,40 @@
+# Scan Docker container images for vulnerabilities
+name: Trivy
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+      - ".circleci/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t docker.io/metabase/metabase:${{ github.sha }} .
+
+      - name: Run Trivy vulnerability scanner
```

Comment:
why don't we do this at the end of https://github.com/metabase/metabase/blob/master/.github/workflows/uberjar.yml?

Reply:
I thought about that, but at the same time, I wanted to make things independently and also not run at every commit.

Maybe we can add

```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```

 on the `uberjar.yml` and that will help on the nodes as well.